reportportal/v5/v5/README.md

```diff
@@ -124,7 +124,7 @@ postgresql:
     enable: false
```

Comment:
would it make sense to describe helm (tiller) installation process including some initial RBAC configuration (if any)

Reply:
To my mind, it doesn't make sense, because Helm/Tiller installation is fully described at section 2 of documentation (for AWS/GCP/Azure/DigitalOcean). In case the user wants very detailed info about Helm they can Google it, because this is not RP specific